atividade03/src/main/java/br/edu/uepb/eda/atividade03/PilhaComLista.java

```diff
@@ -0,0 +1,47 @@
+package br.edu.uepb.eda.atividade03;
+
+public class PilhaComLista implements Pilha_IF {
+    private final int maxLength = 10;
```

Comment:
```suggestion
    private final int MAX_LENGTH = 10;
```

variáveis constante geralmente são escritar em caixa alta